e2e/tests/upgrades/upgrade_test.go

```diff
@@ -606,6 +613,136 @@ func (s *UpgradeTestSuite) TestV7ToV8ChainUpgrade() {
 	})
 }
 
+func (s *UpgradeTestSuite) TestV8ToV8_1ChainUpgrade() {
+	t := s.T()
+
+	ctx := context.Background()
+	relayer, channelA := s.SetupChainsRelayerAndChannel(ctx, func(opts *ibc.CreateChannelOptions) {
+		opts.Version = "{\"fee_version\":\"ics29-1\",\"app_version\":\"ics20-1\"}"
+		opts.DestPortName = "transfer"
+		opts.SourcePortName = "transfer"
+	})
```

Comment:
could import `transfertypes` and use consts here, maybe format the version. I copied this from one of the fee tests directly

Reply:
could maybe move it into `testsuite` as is done for `TransferChannelOptions` if its used by both but fine with either, should fail if something changes.